util.py

```diff
@@ -0,0 +1,12 @@
+# standard libraries
+import logging
+from typing import Any, List
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def chunk_list(input_list: List[Any], chunk_size: int = 100) -> List[List[Any]]:
+    """Chunks a given list into a list of lists of a specified size."""
+    chunks: List[List[Any]] = [input_list[x:x + chunk_size] for x in range(0, len(input_list), chunk_size)]
```

Comment:
Can you add a comment the list got chucked and how many chunks it will be be sending the scores in?

Reply:
Yes, I can add a log message. I don't want to say anything about `Submission`s or scores specifically, because the function is intended for general usage (note the type hint of `Any` for the list elements).

`ScoresOrchestration.send_scores` will indicate how many scores were sent in each batch:
https://github.com/tl-its-umich-edu/placement-exams/blob/e02a124674be9ef0954edd1726a1aaef247c258c/pe/orchestration.py#L195

Reply:
ok

Reply:
Okay, I added a log message; see https://github.com/tl-its-umich-edu/placement-exams/pull/24/commits/68dfdc3f4998bf41ec1d15a010b7eb773e36622c

It'll look something like this (copied from tests): 

`Chunked list of length 250 into 3 list(s) with the following length(s): 100, 100, 50`